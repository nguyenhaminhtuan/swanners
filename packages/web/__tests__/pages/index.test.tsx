import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../pages';

test('render without crashing', () => {
  const { container } = render(<Home />);

  expect(container).toMatchSnapshot();
});
