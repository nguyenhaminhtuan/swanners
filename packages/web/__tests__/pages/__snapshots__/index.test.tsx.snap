// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without crashing 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <div
        class="css-view-1dbjc4n"
        data-focusable="true"
        style="background-color: rgb(0, 0, 255); cursor: pointer; padding: 10px 8px 10px 8px; transition-duration: 0s; transition-property: opacity; user-select: none;"
        tabindex="0"
      >
        <div
          class="css-text-901oao"
          dir="auto"
        >
          Click here
        </div>
      </div>
    </main>
  </div>
</div>
`;
